pyfftw/pyfftw.pyx

```diff
@@ -1408,6 +1436,11 @@ cdef class FFTW:
         copy the returned array.
         '''
 
+        if ortho is None:
+            ortho = self._ortho
+        if normalise_idft is None:
+            normalise_idft = self._normalise_idft
+
         if ortho and normalise_idft:
             raise ValueError('Invalid options: ortho and normalise_idft cannot'
                              ' both be True.')
```

Comment:
Here, `None` does **not** imply `False`.  It implies using whatever value for `ortho` or `_normalise_idft` were set to in the `__cinit__` constructor.  This allows one to create an FFTW object with the desired normalisation behavior.  Once the object has been created, it is never necessary to pass either kwarg to the `__call__` method unless it is desired to override the choices made at the time of the FFTW object creation.

Perhaps there is a better way to achieve the same goal?  It might be a bit clearer to just remove the `ortho` and `_normalise_idft` kwargs from the `__call__` method, but that would involve substantially more refactoring elsewhere in the codebase, as these are occasionally used (e.g. in the numpy interfaces)